en/services/parachute.md

```diff
@@ -0,0 +1,158 @@
+| 65536    | `PARACHUTE_ERROR_FLAGS_LOGGING_ERROR`        | There is an error with the parachute's internal logging system.                                                  |
+| 131072   | `PARACHUTE_ERROR_FLAGS_MODULE_RETIRED`       | This parachute module is retired (i.e. too many deployments).                                                    |
+| 262144   | `PARACHUTE_ERROR_FLAGS_GLOW_WIRE_ERROR`      | There is an error with the parachute glow wire.                                                                  |
+| 524288   | `PARACHUTE_ERROR_FLAGS_OFFBOARD_CONNECTION_ERROR` | There is an error with the MAVLink connection between parachute and offboard computer.                        |
+| 1048576  | `PARACHUTE_ERROR_FLAGS_IMU_CALIBRATION_ERROR`| Parachute's internal IMU calibration failed.                                                                     |
+
+
+#### MAV_CMD_SET_PARACHUTE_ARM
+This command can be used to arm/disarm the various parachute trigger sources. By configuring the arm/disarm setting of the various trigger sources, a user is able to change the behavior of the parachute when it is being used. The operation follows the normal [Command Protocol](https://github.com/mavlink/mavlink-devguide/blob/master/en/services/command.md) rules for command/acknowledgment.
+
+There are two parameters for this command. The first parameter is used to indicate the arm/disarm setting. Setting a bit to "0" indicated disarm and setting a bit to "1" indicates arm. The second parameter is a mask-byte that indicates which trigger source is being configured. The bits of the second parameter are mapped to [PARACHUTE_TRIGGER_FLAGS](#PARACHUTE_TRIGGER_FLAGS). The second parameter allows users to not have to track the status of all trigger sources when you want to set arm/disarm for specific sources. 
+
+For example, the user wants to arm the automatic trigger system and flight controller trigger, and the user wants to disarm the manual and geofence trigger. 
+- The first parameter would be `0b01100`.
+   - Bit 0 for `PARACHUTE_TRIGGER_FLAGS_MANUAL` is set to "0" to disarm.
+   - Bit 1 for `PARACHUTE_TRIGGER_FLAGS_ATS` is set to "1" to arm.
+   - Bit 2 for `PARACHUTE_TRIGGER_FLAGS_FC` is set to "1" to arm.
+   - Bit 4 for `PARACHUTE_TRIGGER_FLAGS_GEOFENCE` is set to "0" to disarm.
+- The second parameter would be `0b11001`.
+   - Since bit 0, 1, 2, and 4 are being configured and no other trigger sources are being configured, all of those bits are set to "1" to indicate that those specific trigger sources need their arm statuses to be updated.
+   - Although bit 3 in the first parameter is set to "0", since bit 3 is set to "0" in the second parameter, the parachute should ignore the bit in the first parameter and not change the arm/disarm status of the trigger source mapped to bit 3.
```

Comment:
```suggestion
   - Doesn't matter what bit 3 in the first parameter is set to , since bit 3 is set to "0" in the second parameter, the parachute should ignore the bit in the first parameter and not change the arm/disarm status of the trigger source mapped to bit 3.
```

Reply:
we could also delete this, or say something like "all other bits, such as bit 3, are ignored"